viewport-dimensions.md

```diff
@@ -16,6 +16,17 @@ and flexible media (including images and video) are combined
 to create website designs that transform depending on the capabilities of the
 user's browser, most prominently the browser's width.
 
+#### Using Media Queries
+
+Since you are not targeting specific screen resolutions in your designs,
+there is also no specific recommended set of media query sizes or
+"breakpoints."
+
+Instead, you should begin with a "mobile first" approach: start designing with the
+narrowest reasonable screen width (300px wide is a recommended starting point)
```

Comment:
Above you mention there is no recommend sizes, then here you recommend 300px. I would suggest either a more vauge statement like "narrowest reasonable screen width or about 300px and then  slowly..." or reference an authoritative source like http://gs.statcounter.com/screen-resolution-stats/mobile/worldwide  (XYZ shows 320px)